convert.py

```diff
@@ -932,6 +932,8 @@ def write_all(fname_out: Path, ftype: GGMLFileType, params: Params, model: LazyM
             elapsed = time.time() - start
             size = ' x '.join(f"{dim:6d}" for dim in lazy_tensor.shape)
             padi = len(str(len(model)))
+            if endianess==gguf.GGUFEndian.BIG:
+                ndarray.byteswap(inplace=True)
```

Comment:
This should be handle in `GGUFWriter.write_tensor_data` just like you do in `add_tensor`. Conversion script should not have no responsibility for handling endianness other than setting it in the constructor.

Reply:
@monatis updated as your comments